content/participants/ttjc@30214391/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Swarnima Shukla
+type: "participant"
+name: "Swarnima Shukla"
+portfolio: "https://swarashukla65.myportfolio.com/"
+github-link: "https://github.com/Swarnimashukla"
+linkedin-link: "https://www.linkedin.com/in/swarnima-shukla-3815b5b8/"
+looking-for: "job of my interest"
+description: "Front-end developer , UX designer , Expert at : HTML | CSS | Javascript ,Learning : Node.js | React , Tools Used : Figma | Invision | Adobe XD , Aspire to become: Full stack developer"
+---
+
+Hi! I am Swarnima Shukla , let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am a final year student pursuing B.tech in computer science and engineering from SRMCEM , Lucknow. I started working on web developement from first year of my college and after developing my first project I decided to take it as a career path. I am winner of Dreambig championship a national level hackathon organized by internshala and first runner up of Automathon , a hackathon organized by Microsoft and Automation anywhere all across country for students and professionals. Being a Beta Microsoft Student Partner I believe in learn , code and teach so I help students learn and work on projects in my technical club .
+
+## why are you taking this challenge?
+
+I am looking for a full time position as Full stack developer in top companies.
+Being a student from tier 3 college and city I never got right kind of guidance to work on something although because I never had guidance so tried my hands in lot of contests , fields and hackathons so here through this challenge I want to learn and work in right direction to get the job and work I have been dreaming of.
+
+## something interesting about you?
+
+I love dancing and sometimes reading novels (developing this habbit). I like to take challenges and sometimes don't think much about how compatible I am because I know I will sail through everything. I am nervous back-stage and  confident when it comes to final peformance .
+
+### help I need?
+
+It would be good if someone tells me how to search for jobs properly as I find most of jobs asking for work experience. Also I need a mock job interview focused on web development.
+
+### how to contact me?
+
+- [Twitter] (https://twitter.com/swarnimashukla5)
```

Comment:
Please use correct syntax for adding a link in markdown. There should be no space between [ ] and () brackets.